src/sentry/integrations/metric_alerts.py

```diff
@@ -229,18 +230,16 @@ def incident_attachment_info(
         except AlertRuleDetector.DoesNotExist:
             raise ValueError("Alert rule detector not found when querying for AlertRuleDetector")
 
+        workflow_engine_params = title_link_params.copy()
+
         try:
             open_period_incident = IncidentGroupOpenPeriod.objects.get(
                 group_open_period_id=metric_issue_context.open_period_identifier
             )
-        except IncidentGroupOpenPeriod.DoesNotExist:
-            raise ValueError(
-                "Incident group open period not found when querying for IncidentGroupOpenPeriod"
-            )
-
-        workflow_engine_params = title_link_params.copy()
-
-        workflow_engine_params["alert"] = str(open_period_incident.incident_identifier)
+            workflow_engine_params["alert"] = str(open_period_incident.incident_identifier)
+        except IncidentGroupOpenPeriod.DoesNotExist as e:
+            sentry_sdk.capture_exception(e)
```

Comment:
Just an FYI, you don't need to call `capture_exception` with an exception in an except block. It should auto-capture the context already.